### PR TITLE
Web: Switch to flow list upon arrival of the first flow.

### PR DESCRIPTION
**Pull request by mhils**
Once we have a flow, we should display that instead of configuration! In particular, `mitmweb -r file` should spawn with the flow list.

### CODE REVIEW

File: web/src/js/ducks/ui/tabs.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export enum Tab {
+    Capture,
+    FlowList,
+    Options,
+    Flow,
+}
+
+const tabsSlice = createSlice({
```
Comment (mhils) [inline web/src/js/ducks/ui/tabs.ts:10]: This here is quite nice really! We should consider using it for more redux stuff going forward.

Comment (lups2000) [reply web/src/js/ducks/ui/tabs.ts:10]: oh that's really nice, so we shouldn't specify anymore stuff like this:
`const TOGGLE_MODE = "TOGGLE_MODE"`, right?

File: web/src/js/components/Header.tsx [hunk 2]
```diff
@@ -4,73 +4,87 @@ import FileMenu from "./Header/FileMenu";
...
                 ))}
                 <HideInStatic>
                     <ConnectionIndicator />
                 </HideInStatic>
             </nav>
-            <div
-                className={classnames({
-                    "empty-header": ActiveMenu === CaptureMenu,
-                })}
-            >
```
Comment (lups2000) [inline web/src/js/components/Header.tsx:73]: you're missing the css class in case of `CaptureMenu` which is different from the other tabs!

**Review by lups2000 — CHANGES_REQUESTED** [review]
The code looks good to me! There is just a small UI problem with `CaptureMenu`
We shouldn't have that space :)
Here the screenshot:

Screenshot 2024-07-05 at 20 44 11

File: web/src/js/ducks/ui/tabs.ts (hunk 1 above)
Comment (mhils) [reply web/src/js/ducks/ui/tabs.ts:10]: yup, much nicer. The old style has benefits (it is more obvious what is actually happening), but I like it nonetheless!

File: web/src/js/components/Header.tsx (hunk 2 above)
Comment (mhils) [reply web/src/js/components/Header.tsx:73]: Missed the updated CSS when staging. Fixed now. :)

**Review by lups2000 — APPROVED** [review]
🍰

**Pull request merged**